nautobot/docs/user-guide/administration/installation/nautobot.md

```diff
@@ -296,7 +303,7 @@ Each of these have their own corresponding setting that defined in `nautobot_con
 
 The `collectstatic` command will create these directories if they do not exist, and in the case of the `static` files directory, it will also copy the appropriate files:
 
-```no-highlight
+```no-highlight title="Collect static files"
 nautobot-server collectstatic
 ```
```

Comment:
This is redundant if the user is running `nautobot-server post_upgrade`, and for consistency that's probably what we should be  recommending here. Not blocking for this PR

Reply:
Yeah, we should look at revamping this section. There are a few things done by post_upgrade. But we still should document the other parts of it.